doc/nrf/device_guides/working_with_nrf/nrf54l/ug_nrf54l15_gs.rst

```diff
@@ -362,35 +77,21 @@ To build and program the sample to the nRF54L15 PDK, complete the following step
 
 1. Connect the nRF54L15 PDK to you computer using the IMCU USB port on the PDK.
 #. Navigate to the :file:`zephyr/samples/hello_world` folder containing the sample.
-#. Build the sample by running the following command::
+#. Build the sample by running the following command
+
+   .. code-block:: console
 
       west build -b nrf54l15dk_nrf54l15_cpuapp@soc1
```

Comment:
I see we updated nrf54l.rst with:
nrf54l15pdk_nrf54l15_cpuapp for PDK revision v0.2.1, AB0-ES7.
nrf54l15pdk_nrf54l15_cpuapp@0.3.0 for PDK revision v0.3.0.

So this "west build -b nrf54l15dk_nrf54l15_cpuapp@soc1" is then no longer correct.

Reply:
Thanks! updated across the document